### ISSUE TITLE
Inability to set check_data_directory_permissions

### ISSUE THREAD

**Issue by jules2689** [nextcloud/all-in-one#2077]
### Steps to reproduce
1. Use Docker for Windows 
2. Set a mount to a local drive (e.g. `docker volume create --driver local --name nextcloud_aio_nextcloud_datadir -o device="/host_mnt/j/nextcloud" -o type="none" -o o="bind"`)
3. Try to install AIO with docker command below  using Caddy as a Reverse Proxy
4. Get complaints about the directory not being 0770 (see extracted error below and logs at the bottom of this issue)
5. Be unable to set check_data_directory_permissions, therefore cannot install

```
2023-02-25 19:31:59 Your data directory is readable by other users.
2023-02-25 19:31:59 Please change the permissions to 0770 so that the directory cannot be listed by other users.
```

This seems to be known behaviour (https://github.com/nextcloud/docker/issues/1188) and as per https://github.com/nextcloud/docker/issues/70 is not fixable - you just need to tell Nextcloud to ignore data dir permissions (`check_data_directory_permissions`)

### Expected behavior 

Can set `check_data_directory_permissions` to false so Windows can install

### Actual behavior 

It cannot install

### Host OS 

Windows 10, Docker for Windows (tried with and without WSL)

#### Nextcloud AIO version 

Nextcloud AIO v4.4.1

#### Current channel 

latest

#### Other valuable info 

**Command:**

`docker run --sig-proxy=false --name nextcloud-aio-mastercontainer --restart always --publish 8080:8080 -e APACHE_PORT=11000 -e APACHE_IP_BINDING=0.0.0.0 --volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config --volume //var/run/docker.sock:/var/run/docker.sock:ro -e NEXTCLOUD_DATADIR="nextcloud_aio_nextcloud_datadir" nextcloud/all-in-one:latest`

**Logs:**

```
2023-02-25 19:31:03 + '[' -f /dev-dri-group-was-added ']'
2023-02-25 19:31:03 ++ find /dev -maxdepth 1 -mindepth 1 -name dri
2023-02-25 19:31:03 + '[' -n '' ']'
2023-02-25 19:31:03 + set +x
2023-02-25 19:31:03 Installing imagemagick via apk...
2023-02-25 19:31:05 Enabling Imagick...
2023-02-25 19:31:06 Configuring Redis as session handler...
2023-02-25 19:31:06 Setting php max children...
2023-02-25 19:31:06 Initializing nextcloud 25.0.3.2 ...
2023-02-25 19:31:22 Initializing finished
2023-02-25 19:31:22 New Nextcloud instance.
2023-02-25 19:31:22 Installing with PostgreSQL database
2023-02-25 19:31:22 Starting Nextcloud installation...
2023-02-25 19:31:58 Nextcloud was successfully installed
2023-02-25 19:31:59 Your data directory is readable by other users.
2023-02-25 19:31:59 Please change the permissions to 0770 so that the directory cannot be listed by other users.
2023-02-25 19:31:59 
2023-02-25 19:31:59 An unhandled exception has been thrown:
2023-02-25 19:31:59 Exception: Environment not properly prepared. in /var/www/html/lib/private/Console/Application.php:166
2023-02-25 19:31:59 Stack trace:
2023-02-25 19:31:59 #0 /var/www/html/console.php(99): OC\Console\Application->loadCommands(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
2023-02-25 19:31:59 #1 /var/www/html/occ(11): require_once('/var/www/html/c...')
2023-02-25 19:32:00 #2 {main}Waiting for appdata to become available...
```

**Comment by szaimen** [comment]
Hi, thanks for the report! I'll fix this with https://github.com/nextcloud/all-in-one/pull/2080 in the next Beta release.

**Comment by jules2689** [comment]
That's great to hear! Is there a cadence for beta releases? I'm psyched to use Nextcloud when this is in.

**Comment by szaimen** [comment]
This is now released with v4.5.1 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Comment by mchenier** [comment]
I used the nextcloud/all-in-one:beta.
I added -e SKIP_DATA_DIRECTORY_PERMISSION_CHECK="yes" in my docker run command. I still have the "Please change the permissions to 0770 so that the directory cannot be listed by other users". My drive is NTFS. I heard that ext4 should fix it but I can't change it.
Anything else to bypass that check ?

**Comment by szaimen** [comment]
You need to set the permissions correctly for smb beforhand. See https://github.com/nextcloud/all-in-one#can-i-use-a-cifssmb-share-as-nextclouds-datadir

**Comment by Danbuth** [comment]
I use the beta version and set `-e SKIP_DATA_DIRECTORY_PERMISSION_CHECK="yes"`, but it  does not work.

**Comment by Danbuth** [comment]
> Hi, thanks for the report! I'll fix this with #2080 in the next Beta release.

I use the beta version and set -e SKIP_DATA_DIRECTORY_PERMISSION_CHECK="yes", but it does not work.
